packages/react/mixins/mixin.server.js

```diff
@@ -9,7 +9,7 @@ const isPlainObject = require('is-plain-object');
 
 const {
   override,
```

Comment:
Please call this `overrideSync` to make the distinction clear - see also..:

https://github.com/untool/untool/blob/382de1056c68a16d0fe03db642ed65d4d0574b53/packages/express/mixins/mixin.core.js#L7-L10